eng/scripts/syncversionclient.ps1

```diff
@@ -91,4 +91,6 @@ function SyncVersionClientFile([String]$GroupId) {
     }
 }
 
+SyncVersionClientFile -GroupId "com.azure"
```

Comment:
@benbp  Is there a way to wrap this in a main function or something that works like a programing language?. Currently if you import the script it runs this too, which I will like to avoid.